### ISSUE TITLE
Show/Find Favorite GeekLists

### ISSUE THREAD

**Issue by bilbothebaggins** [ccomeaux/boardgamegeek4android#87]
Maybe I'm missing something, but:
I would like to be able to view my user's favorite GeekLists, or maybe search for GeekList in the app. As it is, you can only show the single list with the three sorting options, hot/active/... and that's nice for finding some random lists, but not for accessing a known one. And the BGG website is really crap on mobile screen with GeekLists (at least as far as I managed so far), the App shows it so much better.

Having a 4th and 5th sorting/filter option "My GeekLists" and "Favorites" would really be great!

Oh, and thanks for a fantastic App! :-)

**Comment by bilbothebaggins** [comment]
I started looking at the sources: It seems the three categories hot/recent/active are fetched via Json thorugh the BGG url `/geeklist/module?ajax=1&domain=boardgame&nosession=1&showcount=12&tradelists=0&version=v2` plus the sort type. I have not yet found any docs on this (not holding my breath), but as far as I can see there's only these three options this API, so I guess it would have to be fetched differently (if sanely possible at all).

What would be possible though would be to allow to enter and save a list of geeklist IDs in the app, as that's what's fetched in the end. Something like local geeklist bookmarks :-)

**Comment by ccomeaux** [comment]
You pretty much hit the nail on the head. The API we use doesn't expose "My" or "Favorites". In fact, the API we use is private and not something we should probably be using at all. But it's working for now.

I had a similar thought to allow local bookmarking of geeklists, but that's kind of duplicating what BGG offers, so I've been working on other features instead. Hopefully they will make this part of the official API one day. Until then, I'm not likely to act on it. (But I do take pull requests!)

Thanks!

**Comment by bilbothebaggins** [comment]
> local bookmarking of geeklists, but that's kind of duplicating what BGG offers

Yeah, except that much of BGG/web is basically unusable on a smartphone. Shame, really, esp. for something like geeklists, if there was a decent mobile view for geeklists, I wouldn't miss this in the app at all.

Maybe I'll get around to hacking something up, and send in a pull request.

cheers!

**Comment by arnauldvm** [comment]
I was also looking for this feature ("My" geeklists).

Good that you accept pull requests. I've already fiddled with geek lists for a python library project (https://github.com/lcosmin/boardgamegeek). I'll give a try for this app if I find some time.
